examples/guess-the-sketch/src/app.py

```diff
@@ -94,10 +94,17 @@ def index():
 game_round = 3
 embedding_endpoint = 'http://embeddings-api'
 
+sid_to_player_id = {}
+
+@socketio.on('syncSession')
+def handle_sync_session(player_id):
+    logger.debug('Received syncSession from player %s, sid %s', player_id, request.sid)
+    sid_to_player_id[request.sid] = player_id
```

Comment:
Does `request.sid` change if you reload the page?

Reply:
Yes, it doesn't - it's an ephemeral ID for the socket, AIUI. (FWIW I've seen it recycled before so maybe sometimes the socket restores itself, but I've also seen it broken.)